src/cards/a-card.ts

```diff
@@ -15,3 +15,21 @@ const adorable: Card = {
 };
 
 addCard(adorable)
+
+const altruistic: Card = {
+  name: 'ALTRUISTIC',
+  type: 'colourless',
+  pointsA: (inst: CardInstance): number => {
+    const regexp = new RegExp('^\\d+$')
+    if(regexp.test(inst.metadata['gene_pool_size'])) {
+      return +inst.metadata['gene_pool_size']
```

Comment:
God please don't use the prefix `+`.

An explicit increment and return `-1` or save the value in a temporary value, increment, and return is much nicer.

Also why are you testing it with a regex for justt a number string?!?! Just use a frigging number!

Reply:
if I use JSON this is all a lot easier and I can use typing for most of it...